src/platform/forms-system/src/js/definitions/profileContactInfo.js

```diff
@@ -211,7 +211,7 @@ export const profileReviewErrorOverride = ({
 } = {}) => err => {
   if (typeof err === 'string' && err.startsWith(wrapperKey)) {
     return {
-      contactInfoChapterKey,
+      chapterKey: contactInfoChapterKey,
```

Comment:
Unit test for this in #27614 - included in this PR to prevent future merge conflicts